netci.groovy

```diff
@@ -163,5 +163,5 @@ CreateBuildTasks('Windows_NT', 'daily_disablejit', ' "/p:BuildJIT=false"', ' -di
 // CODE STYLE TASKS
 // ----------------
 
-CreateStyleCheckTasks('./jenkins.check_eol.sh', 'ubuntu_check_eol', 'EOL Check')
-CreateStyleCheckTasks('./jenkins.check_copyright.sh', 'ubuntu_check_copyright', 'Copyright Check')
+CreateStyleCheckTasks('./jenkins/check_eol.sh', 'ubuntu_check_eol', 'EOL Check')
+CreateStyleCheckTasks('./jenkins/check_copyright.sh', 'ubuntu_check_copyright', 'Copyright Check')
```

Comment:
Good. For this reason, we expect the checks to fail. We should discuss how to bypass the checks and bring this in. Now might be a good time to add a checkbox to bypass the checks.